### ISSUE TITLE
Error: [ng-i18next] Can't find i18next! - when module i18next is loaded with CommonJS

### ISSUE THREAD

**Issue by jaganathanb** [i18next/ng-i18next#99]
Loading i18next of CommonJS version is not supported in ng-i18next. Hence this error. 
Please provide a patch.

**Comment by jaganathanb** [comment]
This issue is discussed here - https://github.com/i18next/ng-i18next/issues/65

**Comment by jaganathanb** [comment]
Is there any solution to use ng-i18next with Webpack?

**Comment by peebeebee** [comment]
No solution for WebPack and JSPM? 
Why is this closed?

**Comment by anwalkers** [comment]
Reopen and work on fix for WebPack and JSPM.

**Comment by peebeebee** [comment]
For JSPM:

npm does not have the new version of ng-i18next.
I believe I solved this issue by installing i18next, and i18next-xhr-backend from NPM. And ng-i18next from github.

So 
```
jspm install npm:i18next
jspm install npm:i18next-xhr-backend
jspm install github:i18next/ng-i18next
```

**Comment by anwalkers** [comment]
@archer96 @jamuhl I would like to update the npm package to the 1.0.1 version. I reached out to the listed owner and he said that Andre is now the owner. Andre can you add me as an owner? Here is my [npm profile](https://www.npmjs.com/~andywalkers).

**Comment by archer96** [comment]
I have added you as an owner. :smiley: 

```sh
$ npm owner add andywalkers ng-i18next
+ andywalkers (ng-i18next)
```
```sh
$ npm owner ls ng-i18next
andywalkers <andy.walkers@gmail.com>
archer96 <info@andremeyering.de>
```

Regards,
Andre

**Comment by anwalkers** [comment]
I have published 1.0.1 to npm. You should now be able to use it with CommonJS.
